features/console.feature

```diff
@@ -1,7 +1,45 @@
 Feature: The console command
   Most aspects of connection to a server are tested in shell feature.
 
-  Scenario: Opening a local Rails console
+  Scenario: Opening a local Rails console with an irb version between 1.2 and 1.2.6
+    Given the irb version is "1.2"
     When I run `geordi console`
     Then the output should contain "# Opening a local Rails console"
-      And the output should contain "Util.run! bundle exec rails console -e development"
+      And the output should contain "Util.run! bundle exec rails console -e development -- --nomultiline"
+
+
+  Scenario: Opening a local Rails console with irb version >= 1.2.6
```

Comment:
You only need to do the version testing once. Since local is so much simpler, I would suggest to do it only here. However, please to a check for all three cases: below 1.2, between 1.2 and 1.2.6, and from 1.2.6.